src/main/common/components/payment-intention/payment-date.ts

```diff
@@ -23,6 +23,10 @@ export abstract class AbstractPaymentDatePage<Draft> {
   abstract createModelAccessor (): AbstractModelAccessor<Draft, PaymentIntention>
   abstract buildPostSubmissionUri (req: express.Request, res: express.Response): string
 
+  getNotice (): string {
```

Comment:
We would require test case to cover this scenarios when we ain't showing notice content.